db/migrate/20161019004726_create_linked_profiles.rb

```diff
@@ -0,0 +1,16 @@
+class CreateLinkedProfiles < ActiveRecord::Migration
+  def change
+    create_table :linked_profiles do |t|
+      t.references :user, index: true, foreign_key: true
+      t.references :linked_site, index: true, foreign_key: true
+
+      t.integer :external_user_id
```

Comment:
This should be a string... I know, confusing, but some other sites are going to be dipshits and not provide an integer ID.  For example, some sites might have UUIDs like `59208787-86ec-4aef-97ae-cede8e72ecc4` or weird base64 crap like `Mmvofg3gGq0c4w`

We need to be ready to accept this